A/aws_c_common/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "aws_c_common"
+version = v"0.9.3"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/awslabs/aws-c-common.git",
+              "997380c904671f88086847e8dc839bceab3d7821"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/aws-c-common
+
+# Patch for MinGW toolchain
+atomic_patch -p1 "${WORKSPACE}/srcdir/patches/aws-c-common.patch"
+find . -type f -exec sed -i -e 's/Windows.h/windows.h/g' -e 's/Shlwapi.h/shlwapi.h/g' '{}' \;
+
+mkdir build && cd build
+if [[ "${target}" =~ "mingw" ]]; then
+   # Require Windows 7
+   winflags="-D_WIN32_WINNT=0x0601"
+fi
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+	-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+	-DBUILD_TESTING=OFF \
+	-DCMAKE_BUILD_TYPE=Release \
+	${winflags} \
+	..
+cmake --build . -j${nproc} --target install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = Product[
```

Comment:
No products for these two?

Reply:
I don't know yet what they are :slightly_smiling_face: